Pipfile

```diff
@@ -12,6 +12,8 @@ pymdown-extensions = "*"
 mdx-include = "*"
 yamllint = "*"
 mkdocs-markdownextradata-plugin = "*"
+mkdocs-mermaid-plugin = {git = "https://github.com/PeterJCLaw/mkdocs-mermaid-plugin",ref = "python-3"}
```

Comment:
This plugin seems to be hardly used, and we may have to maintain it. I can see that your fork adds python 3 support.

Reply:
I'm definitely open to other ways of rendering inline graphs. I don't particularly _want_ to be maintaining this package, but it's not especially big if we do need to.

Reply:
376c7ecf4e162069c96faeead8b8d68830b17117 improves this, but it's still not on PyPI.